people.html

```diff
@@ -35,10 +35,10 @@
 					'paradigm' : 'http://escience.rpi.edu/ontology/sesf/s2s-core/4/0/HierarchicalSearch',
 					'inputs' : [
 						{'name' : s2sConfOntURI + 'SearchNames', 'enabled' : true, 'widget' : s2sConfOntURI + 'GenericWidget'},
-						{'name' : 'http://a9.com/-/opensearch/extensions/geo/1.0/box', 'enabled' : true, 'widget' : s2sConfOntURI + 'WmsOpenLayersBoundingBox'},
-						{'name' : s2sConfOntURI + 'Organizations', 'enabled' : true, 'widget' : s2sConfOntURI + 'ContextSelect'},
-						{'name' : s2sConfOntURI + 'Communities', 'enabled' : true, 'widget' : s2sConfOntURI + 'ContextSelect'},
-						{'name' : s2sConfOntURI + 'Groups', 'enabled' : true, 'widget' : s2sConfOntURI + 'ContextSelect'},
+						{'name' : s2sConfOntURI + 'box', 'enabled' : false, 'widget' : s2sConfOntURI + 'WmsOpenLayersBoundingBox'},
```

Comment:
Yes, by default they are disabled. Wanted to make it look like the Drupal DIRECTORY as much as posslbe. So the facets go left to right along the top and the result set underneath the facets.